config/vmdb.tmpl.yml

```diff
@@ -344,6 +344,8 @@ workers:
           :poll_method: :escalate
         :ems_metrics_collector_worker_kubernetes:
           :poll_method: :escalate
+          :hawkular_port: 5000
```

Comment:
I feel like this should be made more generic and not have the work "hawkular" in there...perhaps `metrics_port`, `metrics_path`.